app/api-v1/routes/attachment/{id}.js

```diff
@@ -1,8 +1,61 @@
 // eslint-disable-next-line no-unused-vars
+const { BadRequestError, InternalError, NotAcceptableError } = require('../../../utils/errors')
+const { parseAccept } = require('../../../utils/parseAcceptHeader')
+
 module.exports = function (attachmentService) {
   const doc = {
     GET: async function (req, res) {
-      res.status(500).json({ message: 'Not Implemented' })
+      const { validate: uuidValidate } = require('uuid')
+      if (!uuidValidate(req.params.id)) {
+        throw new BadRequestError({ message: 'Id not valid uuid format' })
+      }
```

Comment:
do we need this check? Is the parameter not validated automatically?

Reply:
Jonathan suggested validating IDs 🤷

Reply:
if you remove it does a test fail?

Reply:
sorry by test for invalid I meant, test what happens if no attachment exists for that `id`

Reply:
invalid was the wrong word to use, my bad

Reply:
yeah we should have a `404` case. Also we should handle invalid `uuid` values. What I'm questioning is whether this is done for us by the openapi framework or if we have to do it like this ourselves